website_sale_brand_catalogs/tests/test_catalog.py

```diff
@@ -0,0 +1,27 @@
+from odoo.tests.common import TransactionCase, tagged
+
+from odoo.addons.website.tools import MockRequest
+from odoo.addons.website_sale_brand_catalogs.controllers.portal import WebsiteCatalogs
+
+
+@tagged("post_install", "-at_install")
+class TestCatalog(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.website = self.env.ref("website.default_website")
+        self.brand = self.env["product.brand"].create({"name": "My Brand"})
+        self.test_catalog = [("catalog_ids", "!=", False)]
+
+    def test_01_domain_catalogs(self):
+        catalogs = WebsiteCatalogs.get_domain_catalogs(self)
```

Comment:
You should implement a JS test, this is kind of cheating :smile: 

Take into account tests should emulate flows functionally possible.

Same for the below tests

Reply:
You implemented new JS tests but left old ones